src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -405,8 +402,30 @@ trait KeyStoreBuilder {
   lazy val keyStore: KeyStore = new KeyStoreImpl(keyStoreConfig, secureRandom)
 }
 
+trait ApisBuilder extends ApisBase {
+  object Apis {
+    val Eth = "eth"
+    val Web3 = "web3"
+    val Net = "net"
+    val Personal = "personal"
+    val Daedalus = "daedalus"
```

Comment:
I think we could already remove this option

Reply:
but is ok to leave as it is in this PR.

Reply:
I will present this change in the next task.